charms/istio-ingressgateway/src/charm.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python3
+
+import logging
+from pathlib import Path
+import os
+from kubernetes import client, config
+
+from ops.charm import CharmBase
+from ops.main import main
+from ops.model import ActiveStatus, MaintenanceStatus, WaitingStatus, BlockedStatus
+
+from oci_image import OCIImageResource, OCIImageResourceError
+from k8s_service import RequireK8sService
+
+
+class IstioIngressGatewayCharm(CharmBase):
```

Comment:
More bikeshedding, but can we switch all of these class names to just be `class Operator(CharmBase):`? I don't see much reason to name the class specific to the charm.